data/local_data_source/search/src/test/java/com/sanaa/search/search_history/LocalSearchHistoryDataSourceImplTest.kt

```diff
@@ -0,0 +1,85 @@
+package com.sanaa.search.search_history
+
+import com.sanaa.search.dataSource.local.dto.RecentViewedLocalDto
+import com.sanaa.search.search_history.dao.QueryDao
+import com.sanaa.search.search_history.dao.RecentViewedDao
+import io.mockk.coVerify
+import io.mockk.mockk
+import kotlinx.coroutines.test.runTest
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+class LocalSearchHistoryDataSourceImplTest {
+    private lateinit var dataSource: LocalSearchHistoryDataSourceImpl
+    private lateinit var queryDao: QueryDao
```

Comment:
This is mock, so you don't need to generate an instance before each test, I prefer initial the object here
```suggestion
    private lateinit var queryDao: QueryDao = mockk(relaxed = true)
```